src/gcp/firedata.ts

```diff
@@ -24,28 +24,7 @@ function _handleErrorResponse(response: any): any {
  * @param projectId Project from which you want to get the ruleset.
  * @param instanceName The name for the new Realtime Database instance.
  */
```

Comment:
Is this comment relevant to anything any more? it looks like the one for `listDatabaseInstances` was removed (and it wasn't even right), so maybe the one for that could be corrected (and this "Create a new Realtime Database instance" block removed)

Reply:
Comment updated. Although the wrong comment was removed, since the original comments were identical I just updated this one to "List Realtime Database instances". Hope this is okay.